modifications/keyrotationcontroller.cpp

```diff
@@ -0,0 +1,48 @@
+#include "keyrotationcontroller.h"
+
+KeyRotationController::KeyRotationController(QObject *parent) : QObject(parent)
+{
+    pathOnBoard = "APM";
+    keyFileName = "key.pem";
```

Comment:
The path and key file name could be passed as params into this constructor. And the main params should come from a config.